vmdb/app/models/miq_database.rb

```diff
@@ -100,4 +101,11 @@ def verify_credentials(auth_type=nil, options={})
 
     MiqTask.wait_for_taskid(RegistrationSystem.verify_credentials_queue).task_results if auth_type == :registration
   end
+
+  private
+
+  def registration_server_url_is_valid
+    return if registration_type != "rhn_satellite"  # Only validating Satellite 5 URL
+    errors.add(:registration_server, "expecting http://server.example.com/XMLRPC") unless /^https?:\/\/.+\/XMLRPC$/i === registration_server
```

Comment:
I think you want `=~`, not `===` (although `===` does technically work, its purpose is for case statements)